web/themes/custom/itkdev/itkdev_project_theme/css/project-styles.css

```diff
@@ -5,6 +5,30 @@
     --theme-brand: #047857;
     --theme-brand-hover: #064E3B;
     --theme-border: #E4E4E7;
+    --theme-status: 34 197 94;
+    --theme-warning: 252 211 77;
+    --theme-error: 220 38 38;
+}
+
+.alert-success {
+  --tw-border-opacity: 1;
+  border-color: rgb(var(--theme-status) / var(--tw-border-opacity));
+  --tw-bg-opacity: 0.2;
```

Comment:
I would prefer that we use solid colours, since opacity can give some unexpected results in the ui and for accessibility where contrast is needed. So maybe we should have custom properties like:
--theme-warning-bg
--theme-warning-border
--theme-warning-text
So that we have better control.

Reply:
Also consider setting it up like the brand, primary and secondary, so that we only overwrite colors in the base and do not introduce duplicate class names.